ui/webui/src/components/storage/MountPointMapping.scss

```diff
@@ -13,3 +13,7 @@
 .mount-point-mapping-empty-state-body {
   align-items: center;
 }
+
```

Comment:
Please add a className to the Flex container and make the selector more specific to avoid overwriting rules you don't want. 
Something like className="mount-point-mapping__reformat" would work

Reply:
In the end I don't even need this, because the popup changed to Tooltip.